src/playground/Widgets/Tasks/index.tsx

```diff
@@ -7,16 +8,17 @@ import { forwardRef } from "react"
 
 export interface TasksInsightData {
   title: string
-  tasks: string[]
+  inProgressTasks: string[]
+  dueTasks: string[]
+  noDueTasks: string[]
   overdueLabel: string
-  overdueTasks: number
+  overdueTasksCount: number
```

Comment:
do we need a count? we have the list of the tasks above.

We can to `overdueTasks.length` and etc